src/vg.proto

```diff
@@ -245,6 +257,7 @@ message SnarlTraversal {
 
     // A Snarl with the start and end Visits filled in to label which Snarl these traverse.
     Snarl snarl = 2;
+    string name = 3;
```

Comment:
Are we naming the traversals individually? Or does this refer to the parent snarl?